qml/main.qml

```diff
@@ -1,11 +1,20 @@
 import QtQuick 2.2
 import Sailfish.Silica 1.0
+import QtMultimedia 5.5
 import "pages"
 
 ApplicationWindow
 {
     id: appWindow
 
+    property string coverMode: ""
```

Comment:
Have you ever tried the `State` QML object? Do you think it would help in maintaining the state of the app? Or we could define some set of global functions that is responsible of updating the app state in a controlled way?

Don't know if that makes any sense, I have been studying smth like that in my other app... :)